openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -78,7 +91,7 @@ def collect_instances(self):
         for tool in tools:
             data = self.get_managed_tool_data(tool)
             if not data:
-                data = self._collect_unmanaged_saver(tool)
+                data = self._collect_saver(tool)
```

Comment:
```suggestion
                continue
```

Unmanaged savers should just be ignored. Right?